src/LinkDotNet.Blog.Web/Features/Components/ShortBlogPost.razor

```diff
@@ -43,7 +43,7 @@
 			<h2></h2>
 			<p>@MarkdownConverter.ToMarkupString(BlogPost.ShortDescription)</p>
 			<p class="read-more">
-				<a href="/blogPost/@BlogPost.Id" aria-label="@BlogPost.Title">Read the whole article</a>
+                <a href="/blogPost/@BlogPost.Slug" aria-label="@BlogPost.Title">Read the whole article</a>
```

Comment:
Given the `page` directive:
```csharp
@page "/blogPost/{blogPostId}/{slug?}"
```

This doesn't seem right, does it?

Reply:
You're right. I had considered the blogpostId as a part of the slug in the blogpost slug generator. I fixed it.